container-chains/runtime-templates/frontier/src/lib.rs

```diff
@@ -715,6 +716,66 @@ impl pallet_maintenance_mode::Config for Runtime {
     type XcmExecutionManager = XcmExecutionManager;
 }
 
+#[dynamic_params(RuntimeParameters, pallet_parameters::Parameters::<Runtime>)]
+pub mod dynamic_params {
+    use super::*;
+
+    #[dynamic_pallet_params]
+    #[codec(index = 3)]
+    pub mod contract_deploy_filter {
+        #[codec(index = 0)]
+        pub static AllowedAddressesToCreate: DeployFilter = DeployFilter::All;
+        #[codec(index = 1)]
+        pub static AllowedAddressesToCreateInner: DeployFilter = DeployFilter::All;
+    }
+}
+
+impl pallet_parameters::Config for Runtime {
+    type AdminOrigin = EnsureRoot<AccountId>;
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeParameters = RuntimeParameters;
+    type WeightInfo = ();
```

Comment:
Benches!

Reply:
Yep, I was going to add them after some overall testing :)

Reply:
Added in 6c30206edc42e0f055df800ed4ebff2392b14fcd

Reply:
Added in f3a8a9ae1cd30dd9a8794a2adaa5100584c22759